modules/migrating-control-plane-nodes-to-a-different-storage-domain-rhv.adoc

```diff
@@ -0,0 +1,29 @@
+[id="migrating-control-plane-nodes-to-a-different-storage-domain-rhv"]
+= Migrating control plane nodes to a different storage domain on {rh-virtualization}
+
+{product-title} does not manage control plane nodes, so they are easier to migrate than compute nodes. You can migrate them like any other virtual machine on {rh-virtualization-first}.
+
+Perform this procedure for each node separately.
+
+.Prerequisites
+
+* You are logged into the {rh-virtualization-engine-name}.
+* You have identified the control plane nodes. They are labeled *master* in the {rh-virtualization-engine-name}.
+
+.Procedure
+
+. Select the virtual machine labled *master*.
+
+. Shut down the machine.
+
+. Click the *Disks* tab.
+
+. Click the virtual machine's disk.
+
+. Click *More Actions*{kebab} and select *Move*.
+
+. Select the target storage domain and wait for the move process to complete.
+
+. Start the virtual machine.
+
+. Wait until the {product-title} cluster is stable before moving to the next control plane node. ???HOW DO YOU YOU KNOW IT'S STABLE???
```

Comment:
The node should show up in the output of the following command as Ready.

```
oc get nodes
```

Reply:
OK, take a look now.